dataactcore/README.md

```diff
@@ -6,78 +6,19 @@ The DATA Act Core is a collection of common components used by other DATA Act re
-
-```bash
-$ python setupStaging.py
-$ python setupValidationDB.py
-```
-
-For example: `loadApprop.py` may be run to create the included rule set for testing an appropriations file, or you may replace `appropriationsFields.csv` and `appropriationsRules.csv` with custom versions to run a different set of rules.
-
-If you want to use an updated list of Treasury Account Symbols for the validator checks, you'll need to get the updated [`all_tas_betc.csv`](https://www.sam.fms.treas.gov/SAMPublicApp/all_tas_betc.csv) file and place that in the [scripts/](https://github.com/fedspendingtransparency/data-act-broker-backend/tree/master/dataactvalidator/scripts) folder before running:
-
-```bash
-$ python loadTas.py
-$ python setupTASIndexs.py
-```
-
-Once these scripts have been run, the databases will contain everything they need to validate appropriations files.
-
-### Data Broker Database Reference
-
-After broker setup, there will be five databases:
+The DATA Act broker uses the following databases; the models and setup scripts for all of them live in the DATA Act core project.
 
 * `error_data` - Holds file level errors in the `file_status` table, along with information about number of row level errors of each type in the `error_data` table. A complete list of every separate occurrence can be found in the error report csv file.
 * `job_tracker` - Holds information on all validation and upload jobs, including status of jobs and relations between jobs. The `job_status` table is the main place to get this information and provides file name/type, status of the job, the job's associated submission, and the table in which the results are located. The `job_dependency` table details precedence constraints between jobs, giving the job IDs for both the prerequisite and the dependent job.
 * `staging` - Holds records that passed validation. Each file validated will have a table in this database, named based on the job ID. If the `file_status` table in the `error_data` database lists the file as completed, each record in the input file will be present in either this staging table or the error report.
 * `user_manager` - Holds a mapping between user names and user IDs to be used for providing submission history information to a user.
 * `validation` - Contains all the information a submitted file is validated against. The `file_columns` table details what columns are expected in each file type, and the rule table maps all defined single-field rules to one of the columns specified in `file_columns`. The `multi_field_rule` table stores rules that involve a set of fields, but are still checked against a single record at a time. Finally, the `tas_lookup` table holds the set of valid TAS combinations, taken from the TAS csv file discussed in the setup section.
 
-### How It Works
+## DATA Act Core Project Layout
 
 The DATA Act Core repository is a collection of common components used by other
```

Comment:
We should look through this and other docs for uses of "repository" that are no longer correct.  We could change it to "package".